### PR TITLE
Fix notion config rendering in Airbyte UI

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

Before: 
image

After: 
image

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/airbyte-faros-destination/resources/source-specific-configs.json [hunk 1]
```diff
@@ -555,238 +555,273 @@
...
-                      "default": "Name"
-                    },
-                    "description": {
-                      "type": "string",
-                      "title": "Description",
-                      "description": "The property that determines a project's description",
-                      "default": "Description"
+                  "title": "Projects",
+                  "description": "Configuration for projects",
+                  "oneOf": [
```
Comment (willmarks) [inline destinations/airbyte-faros-destination/resources/source-specific-configs.json:578]: What does wrapping each of these in a oneOf with one item do?

Comment (cjwooo) [reply destinations/airbyte-faros-destination/resources/source-specific-configs.json:578]: https://docs.airbyte.com/connector-development/connector-specification-reference/#using-oneof
`oneOf` tells the Airbyte UI to enclose a set of parameters in a titled box. Without it, all the parameters in an object are flattened, i.e. the `before` picture in the PR description where both task and epic parameters are on the same indent.

**Comment by ypc-faros** [comment]
No code changes needed?

**Review by ypc-faros — APPROVED** [review]

**Review by willmarks — APPROVED** [review]

File: destinations/airbyte-faros-destination/resources/source-specific-configs.json (hunk 1 above)
Comment (willmarks) [reply destinations/airbyte-faros-destination/resources/source-specific-configs.json:578]: Ah nice! thanks for the explanation

**Comment by cjwooo** [comment]
> No code changes needed?

No. The wrapping with `oneOf` is only for Airbyte UI and does not change the schema of the resulting configuration json.

**Pull request merged**